maxim/logger/writer.py

```
@@ -386,6 +388,18 @@ def can_access_filesystem(self):
         except Exception:
             return False
 
+    def process_large_log(self, file_id: str, key: str, serialized_log: str) -> str:
+        """
+        Flush a large commit log to the Maxim API.
+        Exceptions are passed onto the caller.
+        Returns the final gcs command
+        """
+        resp = self.maxim_api.get_upload_url(key, "text/plain", len(serialized_log))
+        self.maxim_api.upload_to_signed_url(
+            resp["url"], serialized_log.encode("utf-8"), "text/plain"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)